locales/ku/LC_MESSAGES/duckduckgo.po

```diff
@@ -1817,43 +1834,45 @@ msgid ""
 "Enter a new passphrase and click/tap %sSave Settings%s. This will save your "
 "data under your new passphrase."
 msgstr ""
+"Borînpeyveke nû têxe û %sSazkariyan Biparêze%s bitikîne. Ev ê daneyên te biparêze "
+"di bin borînpeyva te ya nû."
 
 #. http://i.imgur.com/qo0C6MA.png
 msgctxt "settings"
 msgid "Enter a passphrase"
-msgstr ""
+msgstr "Borînbeyvekê têxe"
 
 msgctxt "maps_directions"
 msgid "Enter start location"
-msgstr ""
+msgstr "Ciheke destpêkê têxe"
 
 #. Placeholder for a field where inputted text will be translated
 msgctxt "translations_module"
 msgid "Enter text"
-msgstr ""
+msgstr "Nivîsê têxe"
 
 #. Alias=pseudônimo: pt_pt
 msgid ""
 "Enter the following details: %sName%s: DuckDuckGo%s URL%s: %s Alias%s: d%s"
-msgstr ""
+msgstr "Hûrgiliyên %sNav%s: DuckDuckGo%s URL%s: %s Nasnav%s: d%s" têxe "
```

Comment:
@CYAXXX There's a `"` sign within the string. Can you please remove it? I think the correct version is:
```suggestion
msgstr "Hûrgiliyên %sNav%s: DuckDuckGo%s URL%s: %s Nasnav%s: d%s"
```